_data/publications.yml

```diff
@@ -156,7 +156,7 @@
 
 - authors: "C. Xu\\*, R. Lopez\\*, E. Mehlman\\*, J. Regier, M.I. Jordan, N. Yosef"
   title: "Probabilistic Harmonization and Annotation of Single-cell Transcriptomics data with Deep Generative Models."
-  details: bioRxiv
+  details: Molecular Systems Biology (in press)
```

Comment:
@romain-lopez can you remove the preprint flag for this entry?

Reply:
ah yes!